src/main/java/cz/smarteon/loxone/app/Control.java

```diff
@@ -98,4 +100,18 @@ protected LoxoneUuids getCompulsoryState(final String stateName) {
         }
     }
 
+    /**
+     * Helper to get the jLocked state UUID for this control.
+     * If the control doesn't support locking, it will return null.
```

Comment:
Consider adding `isLockable` or `supportsLocking` method for convenience (better checking boolean then nullabilty)